BDDSharp/BDDNode.cs

```diff
@@ -1,5 +1,6 @@
-﻿﻿using System;
+﻿using System;
 using System.Collections.Generic;
+using System.Diagnostics;
```

Comment:
Unnecessary import without the debug message I think?

Reply:
I will remove this line